ui/src/app/views/quick-tour-modal/component.tsx

```diff
@@ -0,0 +1,222 @@
+        <br/><br/>
+        When you create your Workspace, you will be prompted
+        to state your research purpose.  For example, when you create a Workspace to study Type
+        2 Diabetes, for research purpose you could enter: “I will use this Workspace to
+        investigate the impact of Geography on use of different medications to treat
+        Type 2Diabetes.”</div>,
+      image: '/assets/images/workspaces.png'
+    },
+    {
+      title: 'Cohorts',
+      content: <div>A “Cohort” is a group of participants you are interested in researching.
+        The Cohort Builder allows you to create and review cohorts and annotate
+        participants in your study group.
+        <br/><br/>
+        For example, you can build a Cohort called “diabetes cases,” to include people
+        who have been diagnosed with type II diabetes, using a combination of billing codes and
+        laboratory values. You can also have a “controls” Cohort. Once you build your cohorts,
+        you can go through and manually review the records for each participant and decide if
+        you want to include or exclude them from your Cohort and make specific
+        annotations/notes to each record.</div>,
+      image: '/assets/images/cohorts.png'
+    },
+    {
+      title: 'Concepts',
+      content: <div>Concepts describe information in a patient’s medical record, such as a condition
+          they have, a  prescription they are taking or their physical measurements. In the Workbench
+          we refer to subject areas such as conditions, drugs, measurements etc. as “domains.” You can
+          search for and save collections of concepts from a particular domain as a “Concept Set.”
+          <br/><br/>
+          For example, if you want to select height, weight and blood pressure information
+          (concepts) from your “diabetes cases” Cohort, you can search for the 3 concepts
+          from the “Measurements” domain and call it “biometrics” Concept Set. You can then
+          use Notebooks to extract that information from your cohort.</div>,
+      image: '/assets/images/concepts.png'
+    },
+    {
+      title: 'Notebooks',
+      content: <div>A Notebook is a computational environment where you can analyze data with basic
+          programming knowledge in R or Python. Several template Notebooks and resources
+          are available within your Workspace that will guide you how to import your
+          Cohort(s) and Concept Set(s) into the Notebook and can assist with basic analyses.
+          <br/><br/>
+         For example, you can launch a Notebook
+          to import your “diabetes cases” Cohort and then select your “biometrics” Concept Set, to
+          get biometrics data for the participants in your Cohort. You can then analyze the data to
+          study correlation between hypertension and diabetes.</div>,
+      image: '/assets/images/notebooks.png'
+    }];
+
+class QuickTourReact extends React.Component<any, any> {
```

Comment:
What do the generics here correspond to?

Reply:
the state and props.  not sure if we should type them or not; was just trying to get my intellij to display less errors